Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -172,6 +154,37 @@ protected void Page_Load(object sender, EventArgs e)
             ResetModulePanes();
         }
 
+        private void InjectThemeJS()
+        {
+            if (Core.Managers.ThemeManager.HasThemeJS())
```

Comment:
Core.Managers.ThemeManager.HasThemeJS() should've been Core.Managers.ThemeManager.Current.HasThemeJS()